build/charts/antrea/templates/agent/daemonset.yaml

```diff
@@ -131,7 +136,7 @@ spec:
           imagePullPolicy: {{ include "antreaAgentImagePullPolicy" . }}
           {{- if ((.Values.testing).coverage) }}
           command: ["/bin/sh"]
-          args: ["-c", "sleep 2; antrea-agent-coverage -test.run=TestBincoverRunMain -test.coverprofile=antrea-agent.cov.out -args-file=/agent-arg-file; while true; do sleep 5 & wait $!; done"]
+          args: ["-c", "antrea-agent-coverage --config /etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=4; while true; do sleep 5 & wait $!; done"]
```

Comment:
I think there is potential to improve readability here.
I would recommend defining a helper script:
```
$ cat run-cov-binary.sh
#!/usr/bin/env bash

# This script is meant to wrap a coverage instrumented binary.
# When the main process is killed, the script will start sleeping until a signal
# is received, providing the opportunity for the user to retrieve code coverage
# files.

"$@"

while true; do
    # Sleep in the background, to allow exiting as soon as a signal is received
    sleep 5 & wait $!
done
```

This script should be included in all coverage container images, and be used as the `ENTRYPOINT`. Then in the YAML manifest, we would just define `args` as follows:
```
args:
  - "antrea-agent-coverage"
  - "--config=/etc/antrea/antrea-agent.conf"
...
```

Reply:
As `ENTRYPOINT`.

I disagree, the manifest looks very messy right now, requiring the invocation to be wrapped with "bash -c", and all the arguments to be passed as a string. Not to mention the while / sleep loop.

The extra layer is not really relevant here IMO: it's a tiny file, in a test image.

Reply:
Done.
Why didn't the CI run for the new commit?

Reply:
You have a conflict, so maybe that's why? Try resolving it and let's see if the jobs run when you push again